tests/specs/test_binstar.py

```diff
@@ -48,7 +48,7 @@ def test_package_without_environment_file(self):
 
     def test_download_environment(self):
         fake_package = {
-            'files': [{'type': 'env', 'version': '1', 'basename': 'environment.yml'}]
+            'files': [{'type': 'env', 'version': '1.0.1', 'basename': 'environment.yml'}]
```

Comment:
FYI, I would leave the old test and create a new test case specifically for the `X.Y.Z` version test case.